devices.js

```diff
@@ -2672,6 +2672,24 @@ const devices = [
             await configureReporting.batteryVoltage(endpoint1);
         },
     },
+    {
+        zigbeeModel: ['Switch 4x EU-LIGHTIFY'],
+        model: 'AB371860355',
+        vendor: 'OSRAM',
+        description: 'Smart+ switch',
+        supports: '4 buttons with click/hold/release',
+        fromZigbee: [
+            fz.osram_lightify_switch_AB371860355_cmdOn,
```

Comment:
Instead of the custom converters, can you use the defaults ones (https://github.com/Koenkk/zigbee-herdsman-converters/blob/master/converters/fromZigbee.js#L634) This will bring it more in sync with how other devices are implemented.

Reply:
I have a Switch 4x EU-LIGHTIFY with model no. AB371860155. This should be added to the model list.

Reply:
> Instead of the custom converters, can you use the defaults ones (https://github.com/Koenkk/zigbee-herdsman-converters/blob/master/converters/fromZigbee.js#L634) This will bring it more in sync with how other devices are implemented.
I'm all in for standardization, but in this case, the OSRAM switch is weird. It sends a "commandOn" when you click top-left and "commandOff" when you press bottom-left, and hue/sat commands when you press the buttons on the right side. Unifying this to <side>_<position>_<activity> totally makes sense in my opinion.

As for AB371860155 model - does it send the same commands?

Reply:
As far as I tested: yes. The broadcast On/Off surprised me a little bit while experimenting with the switch and the manual adjustements to "devices.js" and "fromZIgbee.js". So I did not dig deeper.